05-modifying-containers.qmd

```diff
@@ -258,9 +258,9 @@ Now that you're familiar with the basics of Dockerfiles, let's dive into some mo
 
 Now you are also familiar with `CMD` which runs something when the container is built
 
-> **FROM** creates a layer from the another Docker image.
-> **CMD** specifies what command to run within the container.
-> **RUN** builds your application with make.
+> **FROM** creates a layer from another Docker image.
+> **CMD** specifies the default command to run when a container is started from an image.
+> **RUN** executes commands during the build process of the Docker image.
 > **COPY** adds files from your Docker client’s current directory.
```

Comment:
I'm not sure if this block is being formatted as intended. It currently appears as a code/quoted text block with each line one after the other. I wonder if some kind of bulleted list was intended?